tasks/__init__.py

```diff
@@ -19,7 +19,7 @@
 
 
 try:
-    from tasks import local  # noqa
+    from tasks import local as local_dist # noqa
```

Comment:
👍  I see. We import here just to check file existence without using it. That's why I am not seeing any usage diff.